src/components/references/references.js

```diff
@@ -4,7 +4,60 @@ class References extends React.Component {
   render() {
     return (
       <div>
-        <h3>References</h3>
+        <h2 className='ds-h2' id='references'>References</h2>
+
+        <h3 className='ds-h3'>General References</h3>
+        <ul>
+          <li>
+            Resource Library: <a href='https://qpp.cms.gov/about/resource-library' target='_blank' rel='noopener noreferrer'>qpp.cms.gov/about/resource-library</a>
+          </li>
+          <li>
+            Developer Group for QPP APIs: <a href='https://groups.google.com/forum/#!forum/qpp-apis' target='_blank' rel='noopener noreferrer'>groups.google.com/forum/#!forum/qpp-apis</a>
```

Comment:
should there be a note here about this group not being active anymore?

Reply:
ill ask Michelle